docsite/rst/committer_guidelines.rst

```diff
@@ -0,0 +1,72 @@
+Committers Guidelines (for people with commit rights to Ansible on GitHub)
+``````````````````````````````````````````````````````````````````````````
+
+These are the guidelines for people with commit access to Ansible. Committers are essentially acting as members of the Ansible Core team, although not necessarily as an employee of Ansible and Red Hat. Please read the guidelines before you commit.
+
+These guidelines apply to everyone. At the same time, this ISN’T a process document. So just use good judgement. You’ve been given commit access because we trust your judgement.
+
+That said, use the trust wisely. 
+
+If you abuse the trust and break components and builds, or waste a lot of time asking people to review incomplete or untested pull requests, the trust level falls and you may be asked not to commit or you may lose access to do so.
+
+Features, High Level Design, and Roadmap
+========================================
+
+As a core team member you will be part of the team that actually develops the roadmap! So be engaged and push for what you want. However, Red Hat as a company will commit to certain features, fixes, APIs, etc. for various releases. The company and the Ansible team still has to get those done and out the door. Obligations to users, the community, and customers come first. Because of that, a feature you may want to develop yourself may not get into a release if it impacts a lot of other parts of Ansible.  
```

Comment:
editing suggestion:  As a core team member, you are an integral part of the team that develops the roadmap.  Please be engaged, and push for the features and fixes that you want to see. Also keep in mind that Red Hat, as a company, will commit to certain features, fixes, APIs, etc. for various releases. Red Hat, the company, and the Ansible team must get these committed features (etc.) completed and released as scheduled. Obligations to users, the community, and customers must come first.  Because of thse commitments, a feature you want to develop yourself many not get into a release if it impacts a lot of other parts within Ansible.